### DISCUSSION TITLE
Erro ao tentar inserir campo data e valor

### DISCUSSION THREAD

**Discussion by leufmt** [viniciussanchez/dataset-serialize#128] [Show and tell]
Como nas imagem não consigo utilizar o dataset-serialize para os campos data e valor.
Como resolvo isso?

![image](https://user-images.githubusercontent.com/67557634/147287664-b5b92114-84c9-4bb3-8df9-fd4fb91d60bb.png)

![image](https://user-images.githubusercontent.com/67557634/147287634-2e91ea5a-6a18-412e-8e75-f33c8ef33b6a.png)

**Comment by SidneyDoper** [comment]
Também estou com o mesmo problema, estou utilizando o Delphi 11.
Quando estava utilizando o Delphi 10.4 isso não acontecia.
